ocaml/xapi/context.ml

```diff
@@ -323,7 +325,16 @@ let make ?(http_other_config = []) ?(quiet = false) ?subtask_of ?session_id
             " by task " ^ make_dbg [] "" subtask_of
         )
   ) ;
-  let tracing = start_tracing_helper (parent_of_origin origin) task_name in
+  let span_attributes =
+    [
+      ("xs.xapi.task.id", Ref.really_pretty_and_small task_id)
+    ; ("xs.xapi.task.name", task_name)
+    ; ("xa.xapi.task.uuid", Uuidx.to_string task_uuid)
```

Comment:
Typo: xa/xs

Reply:
Thanks! Here it is https://github.com/xapi-project/xen-api/pull/5368/commits/5213d037487961bf2e26f720509b572e915f7480.